packages/build/src/helpers/user-input.ts

```diff
@@ -0,0 +1,37 @@
+/* istanbul ignore file */
+import readline from 'readline';
+
+export async function ask(prompt: string): Promise<string> {
+  return new Promise((resolve, reject) => {
+    try {
+      const rl = readline.createInterface({
+        input: process.stdin,
+        output: process.stdout
+      });
+
+      rl.question(`${prompt} `, (answer) => {
+        rl.close();
+        resolve(answer);
+      });
+    } catch (e) {
+      reject(e);
+    }
+  });
```

Comment:
```suggestion
  return new Promise(resolve => {
    const rl = readline.createInterface({
      input: process.stdin,
      output: process.stdout
    });

    rl.question(`${prompt} `, (answer) => {
      rl.close();
      resolve(answer);
    });
  });
```

(Exceptions from inside the `Promise` constructor’s callback are turned into rejections automatically :))

Reply:
Huh, that makes sense :D